routes/stats.js

```diff
@@ -88,17 +96,72 @@ statsRouter.get('/month', async (req, res) => {
+    const ouncesTwo = responseTwo.rows[0].ounces_sum != null ? parseFloat(responseTwo.rows[0].ounces_sum) : 0;
     const totalTwo = poundsTwo + ouncesTwo / 16;
-    const rate = ((totalTwo - total) / total).toFixed(2);
+    const rate = (totalTwo != 0 ? ((total - totalTwo) / totalTwo) : total).toFixed(1);
+    res.json(rate);
+  } catch (err) {
+    res.status(400).json(err);
+  }
+});
+
+statsRouter.get('/participants/week', async (req, res) => {
+  try {
+    const response = await pool.query(
+      `SELECT SUM(number_in_party) FROM event_data INNER JOIN events on event_data.event_id=events.id WHERE events.date BETWEEN CURRENT_DATE - INTERVAL '7 days' AND CURRENT_DATE`,
+    );
+    const total = response.rows[0].sum != null ? parseFloat(response.rows[0].sum) : 0;
+
+    const response2 = await pool.query(
+      `SELECT SUM(number_in_party) AS total FROM event_data INNER JOIN events on event_data.event_id=events.id WHERE events.date BETWEEN CURRENT_DATE - INTERVAL '14 days' AND CURRENT_DATE - INTERVAL '7 days'`,
+    );
+   const total2 = response2.rows[0].sum != null ? parseFloat(response2.rows[0].sum) : 0;
+    const rate =  (total2!=0 ? (total-total2)/total2 : total).toFixed(1);
+    res.json(rate);
+  } catch (err) {
+    res.status(400).json(err);
+  }
+});
+
+statsRouter.get('/participants/year', async (req, res) => {
+  try {
+    const response = await pool.query(
+      `SELECT SUM(number_in_party) FROM event_data INNER JOIN events on event_data.event_id=events.id WHERE events.date BETWEEN CURRENT_DATE - INTERVAL '1 year' AND CURRENT_DATE`,
+    );
+    const total = response.rows[0].sum != null ? parseFloat(response.rows[0].sum) : 0;
+
+    const response2 = await pool.query(
+      `SELECT SUM(number_in_party) FROM event_data INNER JOIN events on event_data.event_id=events.id WHERE events.date BETWEEN CURRENT_DATE - INTERVAL '2 years' AND CURRENT_DATE - INTERVAL '1 year'`,
+    );
+    const total2 = response2.rows[0].sum != null ? parseFloat(response2.rows[0].sum) : 0;
+    const rate =  (total2 !=0 ? ((total-total2)/total2) : total).toFixed(1);
+    res.json(rate);
+  } catch (err) {
+    res.status(400).json(err);
+  }
+});
+
+statsRouter.get('/participants/month', async (req, res) => {
+  try {
+    const response = await pool.query(
+      `SELECT SUM(number_in_party) FROM event_data INNER JOIN events on event_data.event_id=events.id WHERE events.date BETWEEN CURRENT_DATE - INTERVAL '1 month' AND CURRENT_DATE`,
+    );
+    const total = response.rows[0].sum != null ? parseFloat(response.rows[0].sum) : 0;
+
+    const response2 = await pool.query(
+      `SELECT SUM(number_in_party) FROM event_data INNER JOIN events on event_data.event_id=events.id WHERE events.date BETWEEN CURRENT_DATE - INTERVAL '3 months' AND CURRENT_DATE - INTERVAL '1 month' `,
+    );
+    const total2 = response2.rows[0].sum != null ? parseFloat(response2.rows[0].sum) : 0;
+    const rate =  (total2!=0 ? (total-total2)/total2 : total).toFixed(1);
```

Comment:
🚫 **[eslint]** <[eqeqeq](https://eslint.org/docs/latest/rules/eqeqeq)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Expected '!==' and instead saw '!='.